configs/blabber.toml

```diff
@@ -54,6 +54,8 @@ start = 1
 topics = 1
 topic_len = 1
 message_len = 64
+# specify an approximate compression ratio for the message payload
```

Comment:
I think it is worth specifying that this is optional and defaults to 1.0, i.e., no compression here.

Reply:
Updated the comments.